database_connection_setup.rb

```diff
@@ -0,0 +1,7 @@
+require './lib/database_connection'
+
+if ENV['ENVIRONMENT'] == 'test'
+  DatabaseConnection.setup('chitter_test')
+else
+  DatabaseConnection.setup('chitter')
```

Comment:
Good names for your databases, I had `chitter_prod` and `chitter_test` - thinking back I think the `_prod` part was redundant.